.eslintrc

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
I don't remember... Think I just do it cuz it's customary / "politically correct" javascript lol.

It's tedious but I'll remove 'em and give it a spin in various envs.